popde/density_estimate.py

```diff
@@ -120,6 +120,73 @@ def prepare_data(self):
         else:
             self.kde_data = self.std_data
 
+
+    def evaluate_with_transf(self, points):
+        """
+        Transforms the input points in the same way as the KDE training data,
+        evaluates the KDE on the transformed points and adjusts the KDE
+        values by the Jacobian of the transformations.
+
+        Parameters:
+        -----------
+        points : array-like, shape (n_samples, n_features)
+            The original parameter space points to be transformed and evaluated.
+
+        Returns:
+        --------
+        kde_vals : array-like, shape (n_samples,)
+        The KDE values adjusted by the Jacobian of the transformations.
+        """
+        if self.input_transf is not None:
+            transf_points = transf.transform_data(points, self.input_transf)
+        else:
+            transf_points = points
+        #normalize
```

Comment:
```suggestion
        # Divide each parameter by the std of the training data
```

Reply:
Note that we have to make _the same_ transformation on the evaluation points as the training points.  So, the standardization step divides each parameter by the std of the training data.  This is just a constant factor.  But then we have to divide the evaluation points by _the same factor_.  Not by the std of the evaluation points (which might even be 0!!) 

So we can use the class std attribute that already exists as it was set in the prepare data method.